### DISCUSSION TITLE
LOG FILES LOCATION

### DISCUSSION THREAD

**Discussion by ricardoxy** [bitcraze/discussions#1231] [Q&A]
Hi,
my question is, lets say you want your flies to fly in a circle pattern and later you want to know the location of each specific fly when it flew in the circle pattern. My guess is that this info can be recorded in the log files, but after the data location is place in the log file, *_*where are the files saved or located_**. In order to retrieve the log files in order to have the flies fly in the same circle pattern as recorded by the log files.

Thank you, any response or suggestion is appreciated.

**Comment by ataffanel** [comment]
The way to log to a file depends a lot of how you are flying: if you are using the client's logging functionalities, the log are saved in the client setting folder. This can be open by clicking the menu settings/open config folder.

If you are flying with a script or with ROS/Crazyswarm then logging to a file should be implemented by your own script. You need to setup a log block for the variable you are interested into and then save the data in a file yourself. The crazyflie lib is not going to do that by itself.
